src/tbb/arena.cpp

```diff
@@ -500,6 +530,8 @@ struct task_arena_impl {
     static int max_concurrency(const d1::task_arena_base*);
     static void enqueue(d1::task&, d1::task_group_context*, d1::task_arena_base*);
     static d1::slot_id execution_slot(const d1::task_arena_base&);
+    static void register_parallel_phase(d1::task_arena_base*);
+    static void unregister_parallel_phase(d1::task_arena_base*, bool);
```

Comment:
It would be better for this internal methods to match the ABI functions, i.e. use `uintptr_t` parameters. This way you will limit the modifications related to these parameters strictly within the methods.

Reply:
Done.